### ISSUE TITLE
LEFT_ALT is triggered with delay every second press

### ISSUE THREAD

**Issue by BioRyajenka** [korlibs/korge#982]
It is funny but on my hardware alt is not being triggered immediately every second press. 
```kotlin
if (Key.values().any { keys[it] }) println("pressed: ${Key.values().filter { keys[it] }}")
```
I.e. if I press alt for the first time, I see `pressed: LEFT_ALT` in console. If I wait any time (tried with 1 secs, 5 secs) and press it again, I won't see the log text immediately. But after like 1 second it will appear. It happens every second press, which is funny.

Maybe worth to mention that I have two alts and both of them are registered as `LEFT_ALT`.

**Hardware:** asus zephyrus g14
**Java version:** azul openjdk 15.0.6
**korgePluginVersion**: 3.0.1

**Comment by BioRyajenka** [comment]
If you hint me when do the engine reads keycodes, I can try to find the cause of the issue myself. (feel like it is related to some specific keycodes)

**Comment by soywiz** [comment]
@BioRyajenka https://github.com/korlibs/korge/blob/e3dc65cb0f83212e847f6bc0e0cb6cdbc35af20a/korgw/src/jvmMain/kotlin/com/soywiz/korgw/awt/BaseAwtGameWindow.kt#L485

**Comment by BioRyajenka** [comment]
So I added logging breakpoint here in `BaseAwtGameWindow.kt`
  <img src="https://user-images.githubusercontent.com/9904421/189885712-4a14cfd0-05a0-4116-9e7b-dd7118d174d6.png" width="45%" />
<img src="https://user-images.githubusercontent.com/9904421/189885621-a0e17e4c-51ac-4a6e-9275-8a4f00386a5f.png" width="45%" />

And when I press ALT I see the following in the log
<img src="https://user-images.githubusercontent.com/9904421/189885850-68b37966-678f-4eea-b3b8-8fcc3cf28bc0.png" width="45%" />

I when I press and release ALT for the first time I see `true 18, false 18`, and when I do the same again I see only `false 18` which means "press" event is not being registered in AWT. I wasn't able to find this problem on the internet so I don't know how it could be fixed and what is the reason. Suppose I will just use another key in my game.

**Comment by idle-code** [comment]
I don't think that you are the only one with that problem: https://stackoverflow.com/questions/51884120/java-awt-alt-press-detected-every-other-occasion

My guess is that Shift/Alt/Ctrl are [modifier keys](https://en.wikipedia.org/wiki/Modifier_key) and can be handled by OS/JVM in peculiar ways.

**Comment by soywiz** [comment]
@idle-code Do you think is it possible to circumvent this problem or fix it?

**Comment by idle-code** [comment]
I have just looked at [KeyEvent](https://docs.oracle.com/en/java/javase/19/docs/api/java.desktop/java/awt/event/KeyEvent.html) documentation and it seems that LEFT_ALT **should be** reported.
And when using an event-based approach it is (I have tested it under Linux):
```kotlin
    override suspend fun SContainer.sceneMain() {
        keys {
            down { println("${it.key} is down") }
            up { println("${it.key} is up") }
        }
    }
```
Auto-repeat still is a problem for other keys but it works with `LEFT_ALT` - events are reported in pairs.

@BioRyajenka could you check if this works for you? Could you provide a stand-alone sample of your code?

Maybe it is the issue with the pooling itself - could it be that some events are missed?

**Comment by BioRyajenka** [comment]
Hey @idle-code thanks for checking,

I receive the following with your code (down events missed):
<details>
  <summary>Log</summary>
  
 LEFT_ALT is down
LEFT_ALT is up
LEFT_ALT is down
LEFT_ALT is up
LEFT_ALT is up
LEFT_ALT is down
LEFT_ALT is up
LEFT_ALT is up
LEFT_ALT is down
LEFT_ALT is up
</details>

> the issue with the pooling
I don't think so. I've [added](https://github.com/korlibs/korge/issues/982#issuecomment-1245257193) debug output in IDEA right at the place where AWT reports events, and it seems AWT misses some of them.

**Comment by BioRyajenka** [comment]
Surprisingly I have some update.

I noticed that mouse move events also stop triggering after I press alt. And then I realised that ALT button turns focus on current window context menu in windows.

So if you press ALT and then arrow down, you will see this. Not sure if it happens on any system or my system only (Windows 10)
![image](https://user-images.githubusercontent.com/9904421/208312683-acf172ac-ea9e-46a5-9176-eb9960a55fa2.png)
